File: server/src/main/java/message/scmessage/game/ScSpawnItem.java
package message.scmessage.game;

import entity.game.item.Item;
import message.scmessage.ScMessage;

public class ScSpawnItem extends ScMessage {
    public String itemID;
    public String itemName;
    public int positionX;
    public int positionY;

    public ScSpawnItem(Item item, int initX, int initY) {
        this.itemID = item.getItemID();
        this.itemName = item.getItemName();
        this.positionX = initX;
        this.positionY = initY;
    }
}

File: Client/Fruitster/Assets/Script/NetConnection/Messages/ScMessages/GamePlay/ScSpawnItem.cs
using System;
using Script.Messages.ScMessages;
using UnityEngine;

namespace Script.NetConnection.Messages.ScMessages.GamePlay
{
    [Serializable]
    public class ScSpawnItem : ScMessage
    {
        public string itemID;
        public string itemName;
        public int positionX;
        public int positionY;
        public override void OnMessage(Session session)
        {
            PlaySceneScript.Instance.SpawnItem(itemID, itemName, new Vector3Int(positionX, positionY, 0));
        }
    }
}